source/manual/howto-convert-an-unencrypted-rds-instance-to-be-encrypted.html.md

```diff
@@ -0,0 +1,103 @@
+---
+owner_slack: "#govuk-platform-engineering"
+title: Convert an unencrypted AWS RDS instance into an encrypted one
+section: Backups
+layout: manual_layout
+parent: "/manual.html"
+---
+
+This playbook describes how to convert an AWS RDS instance which does not have encryption at rest enabled, into an encrypted at rest instance.
+
+These instructions will incur around 1 to 2 hours of downtime. It is possible to do this with only a minute or so downtime, but this requires
+manually setting up logical replication to a secondary RDS instance which is beyond the scope of this guide.
```

Comment:
I think you should mention only the incurred downtime and omit the logical replication part. Alternatively, we could say, "If you want to explore this, then you can contact PE..." etc.

We can discuss the next evolution of RDS encryption, which probably contains logical replication.

What do you think?

Reply:
I'll just leave it and saying it involves downtime.

Generally given the incredibly low risk and low effort of this method, I would much prefer people do it this way if possible anyway. For those DBs which really cannot take a 2 hour downtime we should probably think about architecture anyway :)